README.rst

```diff
@@ -1874,823 +67,14 @@ as in the example below:
-automatically ensure that the macro edge's arguments become part of the expanded query's arguments:
-
-.. code:: python
-
-    query = '''{
-        Animal {
-            name @output(out_name: "animal_name")
-            out_Animal_GrandchildrenCalledNate {
-                uuid @output(out_name: "grandchild_id")
-            }
-        }
-    }'''
-    args = {}
-    schema_with_macros = get_schema_with_macros(macro_registry)
-    expanded_query, new_args = perform_macro_expansion(
-          macro_registry, schema_with_macros, query, args)
-
-    print(expanded_query)
-    # Prints out the following query:
-    # {
-    #     Animal {
-    #         name @output(out_name: "animal_name")
-    #         out_Animal_ParentOf {
-    #             out_Animal_ParentOf @filter(op_name: "name_or_alias", value: ["$nate_name"]) {
-    #                 uuid @output(out_name: "grandchild_id")
-    #             }
-    #         }
-    #     }
-    # }
-
-    print(new_args)
-    # Prints out the following arguments:
-    # {'nate_name': 'Nate'}
-
-Constraints and rules for macro edge definitions
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-- Macro edge definitions cannot use other macros as part of their definition.
-- A macro definition contains exactly one :code:`@macro_edge_definition` and
-  one :code:`@macro_edge_target` directive. These directives can only be used
-  within macro edge definitions.
-- The :code:`@macro_edge_target` cannot be at or within a scope
-  marked :code:`@fold` or :code:`@optional`.
-- The scope marked :code:`@macro_edge_target` cannot immediately contain a type coercion.
-  Instead, place the :code:`@macro_edge_target` directive at the type coercion itself instead of
-  on its enclosing scope.
-- Macros edge definitions cannot contain uses of :code:`@output` or :code:`@output_source`.
-
-
-Constraints and rules for macro edge usage
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-- The :code:`@optional` and :code:`@recurse` directives cannot be used on macro edges.
-- During the process of macro edge expansion, any directives applied on the vertex field belonging
-  to the macro edge are applied to the vertex field marked with :code:`@macro_edge_target` in the
-  macro edge's definition.
-
-In the future, we hope to add support for using :code:`@optional` on macro edges. We have opened
-a `GitHub issue <https://github.com/kensho-technologies/graphql-compiler/issues/586>`_ to track
-this effort, and we welcome contributions!
-
-
-Miscellaneous
--------------
-
-Pretty-Printing GraphQL Queries
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To pretty-print GraphQL queries, use the included pretty-printer:
-
-::
-
-    python -m graphql_compiler.tool <input_file.graphql >output_file.graphql
-
-It's modeled after Python's :code:`json.tool`, reading from stdin and
-writing to stdout.
-
-Expanding :code:`@optional` vertex fields
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I put this section and the one below as sub-sections in the OrientDB page since they only apply to OrientDB. 
https://graphql-compiler.readthedocs.io/en/latest/supported_databases/orientdb.html#performance-penalties

I verified through a git diff that the text was essentially copy pasted.